prac_05/state_names.py

```diff
@@ -3,7 +3,6 @@
 State names in a dictionary
 """
```

Comment:
The code for this one isnt finished, like you havent add codes prompting user input and etc.

Reply:
Reviewer didn't see entire code? Seems finished to me.